### PR TITLE
rise and shine

### PR DESCRIPTION
**Pull request by jkimpai**

### CODE REVIEW

File: breakfast.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+avocado toast
+bagels
```
Comment (ziqi-ma) [inline breakfast.txt:2]: I want bagels

File: breakfast.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+trail mix
+chocolate
+hash browns
+pancakes
+scrambled eggs
+oatmeal
+toast
+waffles
+cereal
+bougie yoghurt parfait
```
Comment (atxu) [inline breakfast.txt:13]: deliciously capitalist

File: breakfast.txt [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+scones
+croissants 
+quiche
+cake
+cold pizza 
+coffee
+smoothies
+milk
+eggs over easy
+sunny side up eggs
```
Comment (endersahin) [inline breakfast.txt:24]: good stuff

File: breakfast.txt [hunk 4]
```diff
@@ -0,0 +1,27 @@
+avocado toast
+bagels
+sour patch
+trail mix
+chocolate
```
Comment (jlkoran) [inline breakfast.txt:5]: I'm not sure this is a breakfast item

**Review by jlkoran — CHANGES_REQUESTED** [review]
Please ensure only appropriate breakfast items are on this list.

File: breakfast.txt [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+cake
+cold pizza 
+coffee
+smoothies
+milk
+eggs over easy
+sunny side up eggs
+poached eggs
+hard boiled eggs
+soft boiled eggs
```
Comment (Quetzalcohuatl) [inline breakfast.txt:27]: option for ostrich eggs? this is a common request

File: breakfast.txt (hunk 1 above)
Comment (coxas) [inline breakfast.txt:2]: option for cream cheese?

Comment (coxas) [inline breakfast.txt:2]: CREAM CHEESE

**Review by coxas — CHANGES_REQUESTED** [review]

**Review by DCorda343 — COMMENTED** [review]
Needs more eggs, but "egg"ceptable.

**Pull request merged**